types/electron-settings/v2/electron-settings-tests.ts

```diff
@@ -1,4 +1,5 @@
-import * as settings from 'electron-settings';
+import settings = require('electron-settings');
+import Observer = require('electron-settings/lib/observer');
```

Comment:
```typescript
observe(keyPath: string, handler: (evt: ElectronSettings.ChangeEvent) => void): Observer;
```
makes Observers available to consumers. Consumers may want to say, keep a list of Observers so they can dispose of them later. This allows them to annotate that list.
```typescript
class MyClass {
    observers: Observer[] = [];
}
```